web/styles/buttons.css

```diff
@@ -0,0 +1,186 @@
+.action-button {
+    /* Reset button styles — borrowed from TailwindCSS Preflight */
+    font-family: inherit;
+    font-feature-settings: inherit;
+    font-variation-settings: inherit;
+    font-size: 100%;
+    margin: 0;
+    text-transform: none;
+    appearance: button;
+    background-image: none;
+    cursor: pointer;
```

Comment:
Does this reset code do something? I don't think we generally want to have unnecessary reset CSS; since it means that style declarations for any specific component that want to override this need to do something.

@karlstolley thoughts?

Reply:
Yeah, this can set up class warfare later on when trying to deviate from things like this with a single class selector. 

If there are specific values, like `font-family`, that we want to make sure that the buttons have, let's explicitly state them. And if there are competing styles of our own doing set on button elements and the like, then let's clean those up at the source, rather than gumming up the new buttons with a bunch of pushback styles. 

(It also makes no sense to me to do things like `font-feature-settings: inherit`, as that is what `font-feature-settings` does anyway, as that property is inherited. But then, I have a very dim view of TailWind.)

Reply:
Agreed and thank you for the insight. Most of these resets were not needed since they were being overridden anyways. The only remaining styles used from these were: `cursor: pointer`, `font-size` and `font-family`, out of which the later two are explicitly set now to `--base-font-size-px` and `Source Sans 3 VF", sans-serif` respectively.